ern-local-cli/src/lib/start.js

```diff
@@ -34,14 +34,16 @@ export default async function start ({
   watchNodeModules = [],
   packageName,
   activityName,
-  bundleId
+  bundleId,
+  extraJsDependencies
 } : {
   miniapps?: Array<string>,
   descriptor?: string,
   watchNodeModules: Array<string>,
   packageName?: string,
   activityName?: string,
-  bundleId?: string
+  bundleId?: string,
+  extraJsDependencies?: Array<DependencyPath>
```

Comment:
Pass Dependency instead of DependencyPath?

Reply:
well .. I guess we will need to review and potentially refactor our `Dependency` vs `DependencyPath` ... Because `DependencyPath` allows for NPM path / file / git path, whereas `Dependency` just basically boils down to `scope/name/version` (so no file / git support).

In the case of `extraJsDependencies`, file and git schemes can be provided for the dependencies, therefore choosing `DependencyPath` over `Dependency`.